pkgs/misc/vim-plugins/default.nix

```diff
@@ -584,6 +584,17 @@ rec {
 
   };
 
+  localvimrc = buildVimPluginFrom2Nix { # created by nix#NixDerivation
```

Comment:
That was generated by using vim-plugin-names-to-nix. There is a github mirror [here](https://github.com/embear/vim-localvimrc), but I don't know how to generate that instead. To be honest, I am not sure how vim2nix finds the sources from the plugin name.